examples/browser/vuejs/connect-two-ae/aepp/src/components/Home.vue

```diff
@@ -202,8 +202,8 @@
   import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message'
 
   // Send wallet connection info to Aepp throug content script
-  const NODE_URL = 'https://sdk-testnet.aepps.com'
-  const NODE_INTERNAL_URL = 'https://sdk-testnet.aepps.com'
+  const TEST_NET_NODE_URL = 'https://sdk-testnet.aepps.com'
```

Comment:
What about `https://testnet.aeternity.io`?

Reply:
Fixed. Thank you